config/routes.rb

```diff
@@ -25,4 +25,8 @@
   get "/admin/paid-orders", to: "admin/orders#index_paid"
   get "/admin/cancelled-orders", to: "admin/orders#index_cancelled"
   get "/admin/completed-orders", to: "admin/orders#index_completed"
+
+  resources :charges
+
+  post 'twilio/connect_customer' => 'twilio#connect_customer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.